docker/puppetdb-postgres/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM postgres:9.6.8
+
+ARG vcs_ref
+ARG build_date
+
+LABEL org.label-schema.maintainer="Puppet Release Team <release@puppet.com>" \
+      org.label-schema.vendor="Puppet" \
+      org.label-schema.url="https://github.com/puppetlabs/puppetdb" \
+      org.label-schema.name="PostgreSQL instance for PuppetDB" \
+      org.label-schema.license="Apache-2.0" \
+      org.label-schema.version="9.6.8" \
+      org.label-schema.vcs-url="https://github.com/puppetlabs/puppetdb" \
+      org.label-schema.vcs-ref=$vcs_ref \
+      org.label-schema.build-date=$build_date \
+      org.label-schema.schema-version="1.0" \
+      org.label-schema.dockerfile="/Dockerfile"
+
+COPY docker-entrypoint-initdb.d /docker-entrypoint-initdb.d
+
+COPY Dockerfile /
```

Comment:
Hmm, OK, so puppetdb's going to have its own postgresql container -- in any case, not sure how they set up that container, but possibly worth noting that the normal default postgresql.conf settings are known to be very conservative (for portability).  If that's also true here, then I'd guess that most installs will want something a bit less conservative.

Reply:
this should be ticketed for investigation

Reply:
tickets.puppetlabs.com/browse/CPR-586